readthedocs/core/views/serve.py

```diff
@@ -269,3 +272,105 @@ def robots_txt(request, project):
         return HttpResponse(open(fullpath).read(), content_type='text/plain')
 
     return HttpResponse('User-agent: *\nAllow: /\n', content_type='text/plain')
+
+
+@map_project_slug
+# TODO: make this cache dependent on the project's slug
```

Comment:
The [docs for `cache_page`](https://docs.djangoproject.com/en/1.11/topics/cache/#django.views.decorators.cache.cache_page) say it is "is keyed off of the URL". Looking at the code, it does look to me like they mean the fully qualified URL including the host so I think we're ok.